src/containers/Admin/Artists/index.js

```diff
@@ -7,92 +7,74 @@ import Header from '../../../components/Header';
 import { Wrapper, FormWrapper } from './styled';
 import { FormControl } from 'react-bootstrap';
 
-class AdminArtists extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            artists: [],
-            show: false,
-            artistName: '',
-            artistUrl: ''
-        };
-      }
+export default function AdminArtists() {
+  const [artists, setArtists] = useState([]);
+  const [show, setShow] = useState(false);
+  const [artistName, setArtistName] = useState('');
+  const [artistUrl, setArtistUrl] = useState('');
 
-    componentDidMount() {
-        fetchAllArtists().then(({data}) => {
-            this.setState({ artists: data })
-        });
-    }
+  useEffect(() => {
```

Comment:
Como isso daqui é um lifecycle do tipo **componentDidMount**, seria o correto utilizar como segundo parâmetro do `useEffect` um `[]`, exemplo:

`useEffect(() => {
    // code to run on component mount
  }, [])`